### PR TITLE
LPS-177325 Fix Buttons of the Management Toolbar on Case History page and popover position when the popover trigger is near the bottom offset Screen

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

**Review by Elvinho — APPROVED** [review]
LGTM 🔥

**Review by samuelleolopes — APPROVED** [review]
Good Job! :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx [hunk 1]
```diff
@@ -30,13 +37,31 @@ type ManagementToolbarFilterProps = {
 	filterSchema?: FilterSchema;
 };
 
-const ManagementToolbarFilter: React.FC<ManagementToolbarFilterProps> = ({
-	filterSchema,
-}) => {
+const FilterBody = ({buttonRef, filterSchema, setPosition}: any) => {
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx:40]: remove any

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx [hunk 2]
```diff
@@ -30,13 +37,31 @@ type ManagementToolbarFilterProps = {
...
+	const [filter, setFilter] = useState('');
+
 	const fields = useMemo(() => filterSchema?.fields as RendererFields[], [
 		filterSchema?.fields,
 	]);
 
+	useEffect(() => {
+		const container = document.querySelector('.testray-page') as any;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx:48]:
```
		const container = document.querySelector('.testray-page');
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx [hunk 3]
```diff
@@ -30,13 +37,31 @@ type ManagementToolbarFilterProps = {
...
+
+		const scrollHandler = () => {
+			const screenHeight = container?.offsetHeight;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx:51]:
```suggestion
			const screenHeight = (container as any)?.offsetHeight;
```

Comment (kevenleone) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx:51]:
```suggestion
			const screenHeight = container!.offsetHeight;
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx [hunk 4]
```diff
@@ -47,9 +72,8 @@ const ManagementToolbarFilter: React.FC<ManagementToolbarFilterProps> = ({
 		return initialValues;
 	}, [fields]);
 
-	const [listViewContext, dispatch] = useContext(ListViewContext);
 	const [fieldOptions, setFieldOptions] = useState<FieldOptions>({});
-	const [filter, setFilter] = useState('');
+	const [listViewContext, dispatch] = useContext(ListViewContext);
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarFilter.tsx:76]: move to after useState

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:relevant

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**